data/hai/hai reveal 1 intro.txt

```diff
@@ -0,0 +1,1002 @@
+		conversation
+			`As you arrive on Hai-home, you find that a reasonable security detachment is waiting at your designated landing berth. The Hai aren't usually ones to be concerned about such things, but you suspect that, as an apparent "war hero", they may be more for your own safety than the Hai's.`
+			`	Word of your status in human space seems to have spread ahead of you, especially considering your apparent reputation on many Hai worlds as one of the few significant human visitors. A small crowd of Hai gather to peek at you on the short walk from the spaceport to the government building you are led to.`
+			branch karma
+				karma > 1
+			`	As you are making the walk you hear questions posed and rumors stated. Occasionally you hear words like "survivor" and "dangerous" thrown in amongst the babble.`
+				goto main
+			label karma
+			`	As you are making the walk you hear questions posed and rumors stated. Occasionally you hear words like "honorable" and "reasonable" thrown in amongst the babble.`
+			label main
+			`	You arrive at a very comfortable office and are received by a mature-looking Hai with the appropriate regalia of a diplomat. She holds out a paw to greet you and says, "My name is Sayari. I was the envoy to the human government a century ago."`
+			`	The present media storm is all the evidence you need that the Republic has never mentioned the Hai, nor publicly acknowledged receiving an envoy from them. If they're in on the secret, then they're probably still trying to work out a suppression strategy and have been too busy to send an envoy quicker than your own trip took.`
+			`	Sayari continues, "Our reports tell us that you are a well-known and respected individual in human space. We also know 'Hai' is quickly becoming a buzzword in your space. A few traders brought the news ahead of you."`
+			`	She pauses and looks at you seriously. "We think it would be prudent for you to represent your people as our point of contact. Would you accept such a role?"`
+			`	You give it a moment of thought.`
+			choice
+				`	"That seems practical."`
+			`	"Fantastic," she says, though her tone is much more business like than the wording would imply. "In that case, if you could please take this to Earth and communicate with your various governments we can begin a dialogue on how best to deal with this current scenario."`
+			choice
+				`	"Various? We only have two governments."`
+			`	"Oh," she says, "we've treated the Syndicate as a de facto government for many decades now; with the amount of influence they have it was the only prudent thing to do."`
+			`	You take the boxed item with you, and an escort awaits to return you to your ship. As the door behind you closes, you hear Sayari mutter to someone, "It really was much easier when they just had one government though. What a mess."`
```

Comment:
I think the best justification for this is not the original one; as became apparent on the recent Korath PR, Hai made contact with humanity by the timeline, immediately after dealing with the Korath raids and subsequently cleaning themselves up to look nice and professional - aggressively diving back into their philosophy of peace.

As far as they were concerned there would've been only one human government at the time, because that would have been a matter of officialdom, and also because pre-oracle Syndicate would have at least been balanced by the Paradise worlds. (Also the Deep is democratically and voluntarily a part of the republic that adopts Republic foreign policy and only seems to care about autonomy in internal affairs.) <- i.e. they were one government for all intents and purposes.

But since then... it's naïve to assume the Hai wouldn't know about the state of things in Human space - they'd just come out of a fresh conflict with the Korath, they'd absolutely put efforts into being aware of their neighbour's status'. Additionally, we know marauders and merchants both manage to get into Hai space at one time or another, and by simple geography, knowledge of the Syndicate's influence will be well-represented in those groups.

The Hai have twenty thousand years of accumulated cultural knowledge and tradition for every thousand years since we developed farming, and thirty thousand for every millennia since writing. With a hundred years of exposure and (as established later in this) good competency with holographics, I imagine the Hai government has a better understanding of the political breakdown in Human space than even some of our own elected officials.